src/lib/Scenes/Artwork/Artwork.tsx

```diff
@@ -9,20 +9,23 @@ import { View } from "react-native"
 import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
 import { ArtworkActionsFragmentContainer as ArtworkActions } from "./Components/ArtworkActions"
 import { ArtworkAvailabilityFragmentContainer as ArtworkAvailability } from "./Components/ArtworkAvailability"
+import { ArtworkTombstoneFragmentContainer as ArtworkTombstone } from "./Components/ArtworkTombstone"
 
 interface Props {
   artwork: Artwork_artwork
 }
 
 export class Artwork extends React.Component<Props> {
   render() {
+    const { artwork } = this.props
     return (
       <View>
         <Flex width="100%" style={{ backgroundColor: "gray" }} height={340} />
         <Flex alignItems="center" mt={2}>
-          <ArtworkActions artwork={this.props.artwork} />
+          <ArtworkActions artwork={artwork} />
+          <ArtworkTombstone artwork={artwork} />
```

Comment:
We need to revise this structure to avoid centering the tombstone and availability

![image](https://user-images.githubusercontent.com/1242537/58168570-77454100-7c86-11e9-80fd-5ee124ae53f6.png)

Should be justified left with padding=2 (i think, from eyeballing the spec)

Reply:
I'm OK with following up on this-- once we finish up this entire top-section maybe we can revisit the layout?

Reply:
Yeah there's no rush :+1: